Automattic/wp-calypso
# [Jetpack new Pricing Page]: add small style fixes for LightBox

#### Proposed Changes
This PR add small fixes to styling:
- Adds compact styling for `TimeFrame` element of price based on threshold 
- Fixes word breaking in options
- Also small refactoring to `useItemPriceCompact` to be able to specify threshold.

#### Testing Instructions

* Run `git fetch && git checkout update/jetpack-product-lightbox-product-data`
  * Run `yarn start-jetpack-cloud`
  * Go to http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1

**OR**

* use the jetpack cloud live link and goto `/pricing?flags=jetpack/pricing-page-rework-v1

* Open lightbox by clicking `More about ...` button on any product card
* Verify that it scales correctly:
**You should not see**

<img width="263" alt="Screenshot 2022-09-15 at 12 09 11" src="https://user-images.githubusercontent.com/60262784/190364854-a164ec71-5819-4297-a66d-98ad76d69242.png">

**You should see changes to the compact price description like this**
<img width="324" alt="Screenshot 2022-09-15 at 12 09 31" src="https://user-images.githubusercontent.com/60262784/190365017-407a4cf0-c179-4960-9c7c-5f4ea4091ff4.png">

#### Pre-merge Checklist






Related to #
